tests/checksums/test_checksums.py

```diff
@@ -0,0 +1,57 @@
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Unit testing of `ensembl.utils.checksums` module."""
+
+from pathlib import Path
+
+import pytest
+from pytest import param
+
+from ensembl.utils.checksums import get_file_hash, validate_file_hash
+
+
+@pytest.mark.dependency()
+def test_get_file_hash(data_dir: Path) -> None:
+    """Tests `get_file_hash()` function.
+
+    Fixtures:
+        data_dir
+    """
+    file_path = data_dir / "file.txt"
+    result = get_file_hash(file_path)
```

Comment:
This should also test the algorithm that is being passed to `get_file_hash`, instead of just the default